honeycomb/core/src/main/java/org/commonjava/o11yphant/honeycomb/HoneycombFilter.java

```diff
@@ -68,6 +69,7 @@ public void doFilter( final ServletRequest request, final ServletResponse respon
             if ( rootSpan != null )
             {
                 rootSpan.addField( "path_info", hsr.getPathInfo() );
+                addHeaderFields( hsr );
```

Comment:
can you add the path_info field by the new method as well?

Reply:
@ruhan1 I'll wrap in another PR